### PR TITLE
Fix monitoring documentation TOC.

### PR DESCRIPTION
**Pull request by daspecster**
Partial resolution for #2246.

The table of contents references were wrong.

### CODE REVIEW

File: docs/json/json/master/toc.json [hunk 1]
```diff
@@ -316,27 +316,19 @@
...
         },
         {
           "title": "Monitored Resource Descriptors",
-          "type": "gcloud/monitoring/index/resourcedescriptor"
+          "type": "gcloud/monitoring/resource/resourcedescriptor"
         },
         {
           "title": "Time Series Query",
-          "type": "gcloud/monitoring/index/query"
+          "type": "gcloud/monitoring/query/query"
```
Comment (dhermes) [inline docs/json/json/master/toc.json]: `query/query` is `{module}/{class}` but case-insensitive? Isn't case-insensitive a bad idea?

**Comment by dhermes** [comment]
@daspecster Let's once and for all write a script for generating this JSON. Hand-crafted JSON is just a bad idea. If we want custom edits, we'll write them in Python and make the script toss them into the autogenerated dictionary.

**Comment by dhermes** [comment]
LGTM

**Comment by daspecster** [comment]
@dhermes yeah, I've started doing this in a branch for another issue. I'll just go full tilt on this for the TOC. I should have a PR for it today.

**Comment by dhermes** [comment]
> for another issue

It should be a standalone anyhow

**Comment by daspecster** [comment]
Yeah, I was thinking of just making the toc a python dict? How does that sound for the first PR?

**Comment by dhermes** [comment]
It doesn't amount to any actual change.

**Pull request merged**